src/vm/h2inc.ps1

```diff
@@ -8,9 +8,17 @@
 # C to MASM include file translator
 # This is replacement for the deprecated h2inc tool that used to be part of VS.
 
+#
+# The use of [console]::WriteLine (instead of Write-Output) is intentional.
+# PowerShell 2.0 (installed by default on Windows 7) wraps lines written with
+# Write-Output at whatever column width is being used by the current terminal,
+# even when output is being redirected to a file. We can't have this behavior
+# because it will cause the generated file to be malformed.
+#
+
 Function ProcessFile($filePath) {
 
-    Write-Output "// File start: $filePath"
```

Comment:
In general `write-output` should never be used to write to the console.  The `write-output` command adds another return value to the pipeline, it just happens that in many cases this gets printed to the console.  If you want to write to the console use `write-host`.

Don't think `Console::WriteLine` is the correct approach here.